pkg/3scale/amp/component/apicast_options.go

```diff
@@ -129,9 +129,6 @@ type ApicastOptions struct {
 
 	ProductionServiceCacheSize *int32
 	StagingServiceCacheSize    *int32
-
-	StagingAdditionalPodAnnotations    map[string]string `validate:"required"`
```

Comment:
same question as above

Reply:
Yes we do allow users to specify annotations that get injected into the Pod but not into the Deployment. The watched secret annotations get injected into the Deployment but not the Pod so there's no risk of them overwriting one another. So we can keep the existing functionality without changes.